au/units/unos.hh

```diff
@@ -31,4 +32,7 @@ struct Unos : UnitProductT<>, UnosLabel<void> {
 };
 constexpr auto unos = QuantityMaker<Unos>{};
 
+namespace symbols {
+constexpr auto U = SymbolFor<Unos>{};
```

Comment:
Interesting... I thought I had googled for units with an abbreviation of "U", and found only "u" (sometimes used for the atomic mass unit).  How did you find this?

Anyway: in the interest of deferring decisions, I'll just revert this unit's symbol.  I could definitely see us supporting enzyme units later on.  Meanwhile, there's no harm in not having a built-in symbol object for unos.

Reply:
>How did you find this?

I googled "unit symbol u". It brought me to https://www.ibiblio.org/units/dictU.html. To be fair, I don't know how widely this unit symbol is used.

> The enzyme unit was adopted by the [International Union of Biochemistry](https://en.wikipedia.org/wiki/International_Union_of_Biochemistry_and_Molecular_Biology) in 1964. Since the [minute](https://en.wikipedia.org/wiki/Minute) is not an [SI](https://en.wikipedia.org/wiki/SI) base unit of time, the enzyme unit is discouraged in favor of the [katal](https://en.wikipedia.org/wiki/Katal), the unit recommended by the [General Conference on Weights and Measures](https://en.wikipedia.org/wiki/General_Conference_on_Weights_and_Measures) in 1978 and officially adopted in 1999.